docs/getting-started/integrations/quick-start.md

```diff
@@ -1,16 +1,16 @@
 ---
 menu-title: Vanilla JS
-meta-title: Vanilla JS CKEditor 5 Installation - Quick Start with npm | CKEditor 5 documentation
-meta-description: Install, integrate and configure CKEditor 5 using Vanilla JS with npm or ZIP.
+meta-title: CKEditor 5 installation - quick start with npm or ZIP | CKEditor 5 documentation
```

Comment:
Why are we removing Vanilla from title, it may affect SEO.

Reply:
Ok, if we explicitly target this keyword, I'll bring it back.